app/components/ads/search-top-leaderboard.js

```diff
@@ -0,0 +1,39 @@
+import Component from '@ember/component';
```

Comment:
What do you think about naming this file `search-ads` instead of `search-top-leaderboard`?